### PR TITLE
KAZOO-5623: Added PostgreSQL support

### PR DESCRIPTION
**Pull request by sergey-safarov**

### CODE REVIEW

File: kamailio/local.cfg [hunk 1]
```diff
@@ -23,6 +23,12 @@
 # # #!trydef REGISTRAR_SYNC_ROLE
 # # #!trydef PRESENCE_NOTIFY_SYNC_ROLE
 
+## PostgreSQL usage - remove all but the last '#' to enable
+# # #!trydef KAMAILIO_DBMS
```
Comment (lazedo) [inline kamailio/local.cfg]: #!substdef already defines KAMAILIO_DBMS, not needed here

File: kamailio/local.cfg [hunk 2]
```diff
@@ -23,6 +23,12 @@
...
+# # #!substdef "!KAMAILIO_DBMS!postgres!g"
+# # #!trydef KAZOO_DB_URL
```
Comment (lazedo) [inline kamailio/local.cfg]: #!substdef already defines KAZOO_DB_URL, not needed here

**Comment by sergey-safarov** [comment]
Luis @lazedo, PR updated

**Comment by lazedo** [comment]
@sergey-safarov this is an amazing addition to the project, thank you.

**Comment by sergey-safarov** [comment]
Hello Luis @lazedo 
I only not resolved this [code](https://github.com/2600hz/kazoo-configs-kamailio/blob/master/kamailio/presence_query-role.cfg#L12-L19).
How to best to it? Via addition regex like
`#!substdef "!KZQ_PRESENCE_SEARCH_SUMMARY_OPTION1! and event = \"\$var(Event)\"!g"`

Or need to use something other way?

**Pull request merged**